ui/cypress/integration/volume/volumelist.spec.js

```diff
@@ -38,7 +38,7 @@ describe('Volume list', () => {
     cy.get('[data-cy="volume_table_name_cell"]')
       .contains('prom-m0-reldev')
       .click();
-    cy.get('@historyPush').should('be.calledOnce').and('be.calledWithExactly', {
+    cy.get('@historyPush').should('be.calledWithExactly', {
```

Comment:
An issue with the stub catching more than one call? Likely caused by the multiple calls to `cy.visit` :wink: